src/styles/blocks/_catalog.scss

```
@@ -0,0 +1,34 @@
+.catalog {
+  display: grid;
+  grid-template-columns: $card-width;
+  justify-content: center;
+  gap: 48px;
+
+  &__card {
+    display: grid;
+    justify-content: center;
+    align-items: center;
+    width: $card-width;
+    height: 300px;
+    border: 1px solid #ccc;
+    border-radius: 5px;
+  }
+
+  @media (min-width: 488px) {
+    & {
+      grid-template-columns: $card-width $card-width;
+    }
```

```suggestion
    grid-template-columns: $card-width $card-width;
```

it will the same but a little shorter